Update Arc Document for LoadBalancerIP/Azure File Storage/extension o… (#127586)
Azure/azure-content: articles/container-apps/azure-arc-overview.md

| Limitation | Details |
|---|---|
| Supported Azure regions | Australia East，Central US, East Asia, East US, North Central US, Southeast Asia, Sweden Central, UK South, West Europe, West US|
| Cluster networking requirement | Must support [LoadBalancer](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) service type |
| Node OS requirement | **Linux** only. | 
| Feature: Managed identities | [Not available](#are-managed-identities-supported) |
| Feature: Pull images from ACR with managed identity | Not available (depends on managed identities) |

> | Feature: Azure File Storage | [SMB driver (>= v1.18.0)](https://github.com/kubernetes-csi/csi-driver-smb) should be installed before using azure file smb storage |

Reviewer: ```suggestion
| Feature: Azure File Storage | [SMB driver (>= v1.18.0)](https://github.com/kubernetes-csi/csi-driver-smb) should be installed before using Azure file SMB storage |
```

- App container console

> [!IMPORTANT]

> > If deploying onto **AKS on Azure Local** ensure that you have [setup HAProxy as your load balancer](/azure/aks/hybrid/configure-load-balancer)  before attempting to install the extension.Additionally, make sure that custom CoreDNS is enabled. 

Reviewer 2: ```suggestion
> If deploying onto **AKS on Azure Local** ensure that you have [setup HAProxy as your load balancer](/azure/aks/hybrid/configure-load-balancer)  before attempting to install the extension. Additionally, make sure that custom CoreDNS is enabled. 
```

> 

> > For guidance on enabling AKS custom CoreDNS, please refer to this [documentation](/cli/azure/containerapp/arc).

Reviewer: ```suggestion
> For information on enabling AKS custom CoreDNS, see the [`az containerapp arc`](/cli/azure/containerapp/arc) CLI documentation.
```

As you create an Azure Container Apps connected environment resource, some subscriptions might see the "No registered resource provider found" error. The error details might include a set of locations and API versions that are considered valid. If this error message is returned, the subscription must be re-registered with the `Microsoft.App` provider. Re-registering the provider has no effect on existing applications or APIs. To re-register, use the Azure CLI to run `az provider register --namespace Microsoft.App --wait`. Then reattempt the connected environment command.

### How can I install SMB Driver?

> You can install the SMB driver using the following Helm command. For additional installation methods, see [Install driver on a Kubernetes cluster](https://github.com/kubernetes-csi/csi-driver-smb/tree/master/charts).

Reviewer 2: ```suggestion
You can install the SMB driver using the following Helm command. For additional installation methods, see [Install driver on a Kubernetes cluster](https://github.com/kubernetes-csi/csi-driver-smb/tree/master/charts).

```

> You can install the SMB driver using the following Helm command. For additional installation methods, please refer to [Install driver on a Kubernetes cluster](https://github.com/kubernetes-csi/csi-driver-smb/tree/master/charts).

Reviewer: ```suggestion
You can install the SMB driver using the following Helm command. For additional installation methods, see [Install driver on a Kubernetes cluster](https://github.com/kubernetes-csi/csi-driver-smb/tree/master/charts).
```
